src/app/components/Preference/PreferenceNumber.js

```diff
@@ -22,10 +23,9 @@ export default class PreferenceInput extends React.PureComponent {
   };
 
   render() {
-    const { value, style, step } = this.props;
+    const { value, style } = this.props;
     return (
       <StyledInput
-        step={step}
```

Comment:
Why was this prop removed?

Reply:
Mostly because I forgot there's an `<input>` `step` property, and it wasn't used anywhere.

Reply:
I didn't know either when this was added, haha 😅 . I think we need to keep this one, it's important for the `line-height` preference.

Reply:
Right now it uses the default value - 1. I restored anyway. :smile: